go/teams/service_helper.go

```diff
@@ -29,29 +29,36 @@ func LoadTeamPlusApplicationKeys(ctx context.Context, g *libkb.GlobalContext, id
 	return team.ExportToTeamPlusApplicationKeys(ctx, keybase1.Time(0), application)
 }
 
-func membersUIDsToUsernames(ctx context.Context, g *libkb.GlobalContext, m keybase1.TeamMembers, forceRepoll bool) (keybase1.TeamMembersDetails, error) {
+func membersUIDsToUsernames(ctx context.Context, g *libkb.GlobalContext, m keybase1.TeamMembers) (keybase1.TeamMembersDetails, error) {
 	var ret keybase1.TeamMembersDetails
 	var err error
-	ret.Owners, err = userVersionsToDetails(ctx, g, m.Owners, forceRepoll)
+	ret.Owners, err = userVersionsToDetails(ctx, g, m.Owners)
 	if err != nil {
 		return ret, err
 	}
-	ret.Admins, err = userVersionsToDetails(ctx, g, m.Admins, forceRepoll)
+	ret.Admins, err = userVersionsToDetails(ctx, g, m.Admins)
 	if err != nil {
 		return ret, err
 	}
-	ret.Writers, err = userVersionsToDetails(ctx, g, m.Writers, forceRepoll)
+	ret.Writers, err = userVersionsToDetails(ctx, g, m.Writers)
 	if err != nil {
 		return ret, err
 	}
-	ret.Readers, err = userVersionsToDetails(ctx, g, m.Readers, forceRepoll)
+	ret.Readers, err = userVersionsToDetails(ctx, g, m.Readers)
 	if err != nil {
 		return ret, err
 	}
 	return ret, nil
 }
 
 func Details(ctx context.Context, g *libkb.GlobalContext, name string, forceRepoll bool) (res keybase1.TeamDetails, err error) {
+	// TODO: forceRepoll is unused since 87b4ec1d699b633a957156108276df154f2fbc2a
```

Comment:
Can you finish out this todo by removing the `forceRepoll` flag and then promoting this comment to a header comment for the `Details` function?

Reply:
Sure, will do! I've already done that but I reverted to keep this PR focused on one thing.